mitosheet/src/mito/utils/actions.tsx

```diff
@@ -1229,7 +1229,8 @@ export const getActions = (
             titleToolbar: 'Graph',
             longTitle: 'Create new graph',
             actionFunction: async () => {
-                await openGraphEditor(setEditorState, sheetDataArray, setUIState, sheetIndex, mitoAPI);
+                const selectedColumnIds = getSelectedColumnIDsWithEntireSelectedColumn(gridState.selections, sheetDataArray[uiState.selectedSheetIndex])
```

Comment:
In Excel, you get the same graph regardless of the order you select the columns. 

ie: if you have column A and B:
1. Select B
2. CMD+Select A
3. Build a graph and it will be the same as vice versa 

In Mito, that is not true because the `getSelectedColumnIDsWithEntireSelectedColumn` returns the columns in the order of the selections. 

Let's add an optional param to `getSelectedColumnIDsWithEntireSelectedColumn` to specify if you want to return the columns in sorted order and then use that here.